db/schema.rb

```diff
@@ -91,6 +90,8 @@
     t.datetime "last_reminder_email_at"
     t.string   "current_project"
     t.text     "pager_number"
+    t.boolean  "prototype",              default: false
```

Comment:
@stephenrichards where has this come from - can't see a migration for it - is it some early prototype left over